runtime/doc/Makefile

```diff
@@ -51,12 +30,6 @@ tags: doctags $(DOCS)
 doctags: doctags.c
 	$(CC) doctags.c -o doctags
 
-manpages: $(MANPAGES)
-
-# OSX groff doesn't support utf-8 as input encoding, so this won't work there.
-.1.man:
-	groff -k -mandoc -Tutf8 $< | sed -e s/.^H//g > $@
```

Comment:
It looks fantastic actually.

On Mac (BSD I guess), there's no `man -l`, but a slash in the path works similarly: `man ./nvim.1`.